js/components/dataset/resource/list.vue

```diff
@@ -155,13 +161,17 @@ export default {
     },
     events: {
         'uploader:progress': function(id, uploaded, total) {
-            const el = this.$el.getElementById(`progress-${id}`);
+            const el = document.getElementById(`progress-${id}`);
```

Comment:
Aren't we breaking the rules of the Vue component? Right now it seems we are looking for something outside of the component itself.

Reply:
No, we are looking for a component inside (defined ine the associated template). Sadly, `getElementById` is not defined on `Element`
In Vue.js 2.0, referencing raw `Element`s for direct access in a loop is supported. Sadly, it's only supported on component in Vue.js 1.x :/